### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by slevenick**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-## 3.14.0 (Unreleased)
+## 3.15.0 (Unreleased)
+## 3.14.0 (March 21, 2020)
```
Comment (rileykarson) [inline CHANGELOG.md]: You'll want to keep `3.14.0` at the top with the "Unreleased" date, it's a very load-bearing part of the release tooling

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+
+FEATURES:
+* **New Data Source:** `google_compute_instance_serial_port` ([#5911](https://github.com/terraform-providers/terraform-provider-google/pull/5911))
+* **New Resource:** `google_compute_region_ssl_certificate` ([#5913](https://github.com/terraform-providers/terraform-provider-google/pull/5913))
+
+IMPROVEMENTS:
+* compute: Add new attribute reference `current_status` ([#5903](https://github.com/terraform-providers/terraform-provider-google/pull/5903))
```
Comment (rileykarson) [inline CHANGELOG.md]: nit: past-tense + what resource is this for?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+* compute: Added `allow_global_access` for to `google_compute_forwarding_rule` resource. ([#5912](https://github.com/terraform-providers/terraform-provider-google/pull/5912))
+* container: added `node_pool.upgrade_settings` (TPG only) ([#5910](https://github.com/terraform-providers/terraform-provider-google/pull/5910))
```
Comment (rileykarson) [inline CHANGELOG.md]: & can you specify the resource?

```suggestion
* container: added `node_pool.upgrade_settings` ([#5910](https://github.com/terraform-providers/terraform-provider-google/pull/5910))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,15 @@
 ## 3.14.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_compute_instance_serial_port` ([#5911](https://github.com/terraform-providers/terraform-provider-google/pull/5911))
+* **New Resource:** `google_compute_region_ssl_certificate` ([#5913](https://github.com/terraform-providers/terraform-provider-google/pull/5913))
+
+IMPROVEMENTS:
+* compute: Add new attribute reference `current_status` ([#5903](https://github.com/terraform-providers/terraform-provider-google/pull/5903))
+* compute: Added `allow_global_access` for to `google_compute_forwarding_rule` resource. ([#5912](https://github.com/terraform-providers/terraform-provider-google/pull/5912))
```
Comment (danawillow) [inline CHANGELOG.md]:
```suggestion
* compute: Added `allow_global_access` to `google_compute_forwarding_rule` resource. ([#5912](https://github.com/terraform-providers/terraform-provider-google/pull/5912))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,15 @@
...
+* container: added `node_pool.upgrade_settings` ([#5910](https://github.com/terraform-providers/terraform-provider-google/pull/5910))
```
Comment (danawillow) [inline CHANGELOG.md]: Can you clarify whether this is to the `node_pool` block on `google_container_cluster` or a top level field on `google_container_node_pool` (or both)?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,17 @@
...
+
+FEATURES:
+* **New Data Source:** `google_compute_instance_serial_port` ([#5911](https://github.com/terraform-providers/terraform-provider-google/pull/5911))
+* **New Resource:** `google_compute_region_ssl_certificate` ([#5913](https://github.com/terraform-providers/terraform-provider-google/pull/5913))
+
+IMPROVEMENTS:
+* compute: Added new attribute reference `current_status` to the `google_compute_instance` resource ([#5903](https://github.com/terraform-providers/terraform-provider-google/pull/5903))
+* compute: Added `allow_global_access` to `google_compute_forwarding_rule` resource. ([#5912](https://github.com/terraform-providers/terraform-provider-google/pull/5912))
+* container: Added `dns_cache_config` field to `google_container_cluster` resource ([#5887](https://github.com/terraform-providers/terraform-provider-google/pull/5887))
+* container: Added `node_pool.upgrade_settings` to `google_container_node_pool` resource ([#5910](https://github.com/terraform-providers/terraform-provider-google/pull/5910))
```
Comment (danawillow) [inline CHANGELOG.md]:
```suggestion
* container: Added `upgrade_settings` to `google_container_node_pool` resource ([#5910](https://github.com/terraform-providers/terraform-provider-google/pull/5910))
```

**Pull request merged**